deps/Qhull/Qhull.cmake

```diff
@@ -1,9 +1,15 @@
 include(GNUInstallDirs)
+
+if(LINUX AND CMAKE_BUILD_TYPE STREQUAL "Debug") # compiling in debug mode on linux causes errors
```

Comment:
it's not a good idea to mix debug with release/relwithdebinfo.
Should we treat it as a separate issue instead of change it to relwithdebinfo silently?